compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/SyntaxInfo.java

```diff
@@ -33,6 +33,8 @@ public class SyntaxInfo {
 
     private static final List<String> BALLERINA_KEYWORDS = Arrays.stream(SyntaxKind.values())
             .filter(syntaxKind -> SyntaxKind.RE_KEYWORD.compareTo(syntaxKind) > 0)
+            .filter(syntaxKind -> SyntaxKind.GROUP_KEYWORD.compareTo(syntaxKind) > 0)
+            .filter(syntaxKind -> SyntaxKind.COLLECT_KEYWORD.compareTo(syntaxKind) > 0)
```

Comment:
```suggestion
            .filter(syntaxKind -> SyntaxKind.COLLECT_KEYWORD.compareTo(syntaxKind) > 0)
```

Reply:
Sorry this should be
`.filter(syntaxKind -> SyntaxKind.RE_KEYWORD.compareTo(syntaxKind) > 0)`